bigchaindb/web/server.py

```diff
@@ -79,8 +80,8 @@ def create_server(settings):
     if not settings.get('threads'):
         settings['threads'] = (multiprocessing.cpu_count() * 2) + 1
 
-    debug = settings.pop('debug', False)
-    app = create_app(debug)
+    app = create_app(settings)
+    settings.pop('debug', False)
```

Comment:
> using the EAFP approach:

Usually I like the EAFP approach, but in this case I just used the `pop` method with a default value to avoid the exception.


> just out of curiosity what is the reason to pop this?

It's to clean up the dictionary before passing it to `StandaloneApplication`, but I can remove it since `load_config` takes care of ignoring keys that are not expected.